pkg/cni/install.go

```diff
@@ -80,7 +80,7 @@ func (i *Installer) Stop() {
 	if i.Mode == constants.AdsMode || i.Mode == constants.WorkloadMode {
 		log.Info("start remove CNI config\n")
 		if err := i.removeCniConfig(); err != nil {
-			log.Error("remove CNI config failed, please remove manual")
+			log.Errorf("err:%v, remove CNI config failed, please remove manual", err)
```

Comment:
In line 81 of the log, a newline character was added. Does it need to be added here? What is the standard for adding or not adding newline characters in Kmesh logs?

Reply:
we donot need to add \n by hand, becasue  log.Errorf/Info/Debug has also added \n implicitly